builder/amazon/common/step_deregister_ami.go

```diff
@@ -19,12 +20,25 @@ func (s *StepDeregisterAMI) Run(state multistep.StateBag) multistep.StepAction {
 	ui := state.Get("ui").(packer.Ui)
 
 	// check for force deregister
+	// owner-alias
 	if s.ForceDeregister {
-		resp, err := ec2conn.DescribeImages(&ec2.DescribeImagesInput{
+		images_imput := &ec2.DescribeImagesInput{
 			Filters: []*ec2.Filter{&ec2.Filter{
 				Name:   aws.String("name"),
 				Values: []*string{aws.String(s.AMIName)},
-			}}})
+			}}}
+
+		if len(s.ForceDeregisterOwners) > 0 {
```

Comment:
If there is an existing test that covers the `force_deregister` option can you copy it and add a case for the new behavior?

If not can you supply an example of a build that would conflict with the old logic? For example any build that has a name conflict with a public image.